marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/artifacts/suiteSetup.sjs

```diff
@@ -0,0 +1,35 @@
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
+const dataHub = DataHubSingleton.instance();
+
+const permissions = [xdmp.permission('data-hub-entity-model-writer', 'update'), xdmp.permission('data-hub-entity-model-reader', 'read')];
+
+dataHub.hubUtils.writeDocument('/entities/TestEntity-NoMappingConfig', {
```

Comment:
You don't need to do this - just toss the content into a file under ./test-data/entities in the test file directory, and then call hub-test:load-entities($test:__CALLER_FILE__) (search for references to that for an example). 

Though I did notice you're writing to staging - any reason for that? We usually run the unit tests against the final app server.